sw/device/silicon_creator/lib/cert/dice_cwt.c

```diff
@@ -57,7 +57,11 @@ static ecdsa_p256_signature_t curr_tbs_signature = {.r = {0}, .s = {0}};
 #define CWT_PROFILE_NAME "android.16"
 
 // Debug=2, Normal=1
-static uint8_t get_chip_mode(void) {
+static uint8_t get_chip_mode_cdi0(void) {
+  return (lifecycle_is_prod() ? 1 : 2);
+}
+
+static uint8_t get_chip_mode_cdi1(void) {
```

Comment:
Could you drop a comment here to explain why there are two seemingly redundant functions?